docs/_pages/subjectguide/biology.md

```diff
@@ -0,0 +1,136 @@
+---
+permalink: /subject-guide/biology/
+title: RSE Toolkit - Subject Guide
+breadcrumb: Subject Guide
+layout: single
+---
+
+# Biology
+
+## Bioinformatics
+
+Major languages:
+
+for tools and also web-development efforts
```

Comment:
Typo: ~~for tools and also web-development efforts~~ For tools and also web development efforts: